content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,197 @@
+
+<Video 
+  source={{
+    webm: 'blog/release-post/4.0/Batch_Changes_Side_by_Side',
+    mp4: 'blog/release-post/4.0/Batch_Changes_Side_by_Side'
+  }}
+  loop={true}
+  title="Server-side Batch Changes"
+  caption="Side by side comparison of deploying Batch Changes locally vs. server-side"
+/>
+
+[Docs](https://docs.sourcegraph.com/batch_changes/explanations/server_side)
+
+
+### ☁️ Dedicated Sourcegraph Cloud instances for enterprise
+
+<Badge text="Admin" color="violet" size="small" onClick={() => window.open("URL_GOES_HERE", "_self")} />
+
+#### Get started with Sourcegraph faster than ever
+After many years of building Sourcegraph into a secure and highly-scalable solution, we're ready to bring it to the cloud. Sourcegraph Cloud now offers dedicated, single-tenant instances of Sourcegraph, making it easier than ever for new teams to get started. You can read more in our announcement blog here[/blog/enterprise-cloud].
+
+[Docs](https://docs.sourcegraph.com/cloud)
+
+### 💖 Advanced admin capabilities
+
+<Badge text="Admin" color="violet" size="small" onClick={() => window.open("URL_GOES_HERE", "_self")} />
+
+#### Save time upgrading to Sourcegraph 4.0 with multi-version upgrades
+Multi-version upgrades help to reduce the burden and time required to upgrade to the latest version of Sourcegraph. Customers will now be able to seamlessly upgrade from any 3.20+ version to Sourcegraph 4.0 in just a few simple steps. With faster upgrades, customers can quickly start using the new features and enhancements that are released every month. 
+
+<Video 
+  source={{
+    webm: 'blog/release-post/4.0/Multi-version%20Upgrades',
+    mp4: 'blog/release-post/4.0/Multi-version%20Upgrades'
+  }}
+  loop={true}
+  title="Multi-version upgrades"
+  caption="Demo of multi-version upgrades"
+/>
+
+[Docs](https://docs.sourcegraph.com/admin/updates#multi-version-upgrades)
+
+<Badge text="Admin" color="violet" size="small" onClick={() => window.open("URL_GOES_HERE", "_self")} />
+
+#### View usage and measure the value of our platform with new and enhanced in-product analytics
+Our new analytics, introduced with version 3.42, make it easier for admins to understand user engagement, measure efficiency in terms of time saved, and help calculate the value of utilizing Sourcegraph and its features. In-product analytics now include ROI-based metrics such as projected value and dev-time-saved for all Sourcegraph features, and individual user usage. Admins can access the analytics, configure the date range from one week to three months, and export the details. 
```

Comment:
I don't think we need to reference when this was introduced. Think of this as the introduction of in-product analytics. We should also highlight use cases + features (it works with every part of the product, its customizable, etc.). We can pull from the blog.